### PR TITLE
initial repository and solution

### PR DESCRIPTION
**Pull request by anmeng10101**

### CODE REVIEW

File: src/Microsoft.Content.Build.Java2Yaml.Constants/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        public const string Src = @"src\";
+        public const string PackageFolder = @"packages";
+
+        public static class BuildTool
+        {
+            const string Maven = "pom.xml";
+        }
+
+            public static class JavadocArgumentModel
```
Comment (daxianji007) [inline src/Microsoft.Content.Build.Java2Yaml.Constants/Constants.cs]: indent?

**Review by daxianji007 — APPROVED** [review]

File: src/java2yaml/Program.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                Console.Error.WriteLine("Unrecognized parameters. Usage : Java2Yaml.exe [code2yaml.json, repo.json]");
+                return false;
+            }
+
+            string configPath = args.Length == 0 ? Constants.ConfigFileName : args[0];
+            string repoListPath = args.Length == 0 ? Constants.RepoListFileName : args[1];
+
+            if (!File.Exists(configPath) || !File.Exists(repoListPath))
+            {
+                Console.Error.WriteLine($"Cannot find config file: {Constants.ConfigFileName} or {Constants.RepoListFileName}");
```
Comment (daxianji007) [inline src/java2yaml/Program.cs]: Constants.ConfigFileName => configPath

File: src/Microsoft.Content.Build.Java2Yaml.Constants/Constants.cs (hunk 1 above)
Comment (superyyrrzz) [inline src/Microsoft.Content.Build.Java2Yaml.Constants/Constants.cs]: style: dedent

**Comment by superyyrrzz** [comment]
What about use single csproj?

**Review by superyyrrzz — APPROVED** [review]

File: src/shared/base.props [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (superyyrrzz) [inline src/shared/base.props:1]: If we need not share config among csprojs, we can embed this directly into csproj, and abandon this file.

File: src/java2yaml/Program.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
+﻿namespace Microsoft.Content.Build.JavaYaml
+{
+    using System;
```
Comment (superyyrrzz) [inline src/java2yaml/Program.cs:3]: docfx v2 has a conversion to put `using` inside `namespace`. It is different from the default behavior of Visual Studio. v3 has put it outside namespace for easier maintenance.

**Pull request merged**